### ISSUE TITLE
[UST-229] [Backend] [Feat]: Enable search post by text

### ISSUE THREAD

**Issue by ryanycw** [social-tw/social-tw-website#544]
**Describe the solution you'd like**
* Refactor the current get post route to allow passing keywords as query params
https://github.com/social-tw/social-tw-website/blob/9cc927d0b2cebc0d550c3463265bc8dc16a0ee05/packages/relay/src/routes/postRoute.ts#L18

* And in fetchPosts search for content that contain keywords 
https://github.com/social-tw/social-tw-website/blob/9cc927d0b2cebc0d550c3463265bc8dc16a0ee05/packages/relay/src/services/PostService.ts#L143
   ```
   if (keywords) {
       whereClause.content = { contains: keywords }
   }
   ```
* Add test in post.test for fetchPosts with keyword 
https://github.com/social-tw/social-tw-website/blob/main/packages/relay/test/post.test.ts

- Story Point: 3
- Reporter: @ryanycw

**Comment by Lisooo790926** [comment]
I think it's related to this issue https://github.com/social-tw/social-tw-website/issues/520, I will wait for the reply then implement it, thanks

**Comment by ryanycw** [comment]
Confirming on this end, so this issue is also resolved with #577, is that correct?

**Comment by ryanycw** [comment]
Closed by #577
